GOVERNANCE.md

```diff
@@ -145,7 +145,7 @@ The ex-member is
 - removed from the list of [team members](#team-members). Ideally by sending a PR of their own, at least approving said PR. In case of forced removal, no approval is needed.
 - removed from the projects. Optionally, they can retain maintainership of one or more repositories if the [team](#team-members) agrees.
 - removed from the team mailing list and demoted to a normal member of the other mailing lists.
-- not allowed to call themselves an active team member any more, nor allowed to imply this to be the case.
+- not allowed to call themselves an active team member anymore, nor allowed to imply this to be the case.
```

Comment:
This file is shared between different repos, so the change would need to be done in other repos. Could revert this change for now (governance team will care of it eventually)? :pray: